components/ApplicantTable.tsx

```diff
@@ -0,0 +1,25 @@
+"use client"
+
+import { Applicant } from "@prisma/client"
+import { createColumnHelper } from "@tanstack/react-table"
+import TanstackTable from "./TanstackTable"
+
+type ApplicantRow = Pick<Applicant, 'cid' | 'firstName'>
+const columnHelper = createColumnHelper<ApplicantRow>()
+
+const columns = [
+  columnHelper.accessor('cid', {
+    cell: (info) => info.getValue(),
+  }),
+  columnHelper.accessor('firstName', {
+    cell: (info) => info.getValue(),
+  })
+]
+
+const ApplicantTable = ({ applicants }: { applicants: ApplicantRow[] }) => {
+    return (
+        <TanstackTable data={applicants} columns={columns} />
+    )
+}
```

Comment:
I would extract the props interface and use the parameterized FC type for the component (in general, I would stick to this style across the codebase). Remember to import FC. Also, the return of the functional component can be implicit when there's no other logic involved.

```suggestion

interface ApplicantTableProps {
    applicants: ApplicantRow[]
}

const ApplicantTable: FC<ApplicantTableProps>  = ({ applicants }) => (
    <TanstackTable data={applicants} columns={columns} />
)
```

Reply:
Resolved in [8918a16](https://github.com/imperial/ug-admissions/pull/2/commits/8918a16e73a190d6a940009a0e43c813bd3f7e9f)